### PR TITLE
Ref: support matic deposit/withdrawal (Coinbase)

### PR DESCRIPTION
**Pull request by cmgustavo**
Screen Shot 2022-10-18 at 10 25 33

### CODE REVIEW

File: src/navigation/wallet/screens/GlobalSelect.tsx [hunk 1]
```diff
@@ -263,9 +264,8 @@ const GlobalSelect: React.FC<GlobalSelectProps> = ({
     if (recipient?.currency) {
       wallets = wallets.filter(
         wallet =>
-          wallet.currencyAbbreviation === recipient?.currency ||
-          (recipient?.opts?.showERC20Tokens &&
-            IsERCToken(wallet.currencyAbbreviation, wallet.chain)),
+          wallet.currencyAbbreviation === recipient?.currency &&
+          wallet.chain === recipient?.chain,
```
Comment (rastajpa) [inline src/navigation/wallet/screens/GlobalSelect.tsx:268]: this condition could be breaking cases in which the chain does not exist

**Review by rastajpa — DISMISSED** [review]
LGTM, withdraw and deposit made successfully

File: src/navigation/wallet/screens/GlobalSelect.tsx (hunk 1 above)
Comment (cmgustavo) [reply src/navigation/wallet/screens/GlobalSelect.tsx:269]: Thanks! Just added another condition to avoid crash if `chain` no exists.

**Review by rastajpa — DISMISSED** [review]
ACK

**Review by rastajpa — DISMISSED** [review]

**Review by rastajpa — APPROVED** [review]

**Review by rastajpa — APPROVED** [review]
ACK

**Pull request merged**